src/video/openvr/openvr_capi.h

```diff
@@ -46,21 +46,15 @@
 	#endif // OPENVR_API_EXPORTS
 #endif
 
+#ifndef USE_SDL
 #include <stdint.h>
 
 #if defined( __WIN32 )
 typedef char bool;
 #else
 #include <stdbool.h>
 #endif
-
```

Comment:
Please don't remove these types. Remember, this is an autogenerated file and we want to minimize changes.

Reply:
I see. Those typedefs are duplicates though (and one is even a triplicate.) If I don't touch them, I should only do the USE_SDL change and the SDL video struct typedef changes??

Reply:
Are you actually getting warnings or errors building with them? If so, then feel free to commit as-is. Otherwise, let's not do maintenance cleanup here.

Reply:
If you're getting errors, then we should take your changes.

Reply:
OK then, will merge once the CI is green.

P.S.: Do you know why SDL_openvrvideo.h overrides and defines `GL_APIENTRY` and `GL_APICALL` as nothing for windows??? That feels wrong, but I may very well be missing something.